configman/converters.py

```diff
@@ -244,6 +243,11 @@ def str_to_classes_in_namespaces(
                               class.
                               """
 
+    # these are only used within this method.  No need to pollute the module
+    # scope with them
```

Comment:
Is it really polluting?
I generally don't do function-level imports unless it's to avoid circular imports.